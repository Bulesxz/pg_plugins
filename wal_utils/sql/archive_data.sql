-- Sanity check for archive_get_size
SELECT archive_get_size('../no_parent'); -- error
SELECT archive_get_size('/no_absolute'); -- error
