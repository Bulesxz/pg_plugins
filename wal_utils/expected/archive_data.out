-- Sanity check for archive_get_size
SELECT archive_get_size('../no_parent'); -- error
ERROR:  reference to parent directory ("..") not allowed
SELECT archive_get_size('/no_absolute'); -- error
ERROR:  absolute path not allowed
